src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_COST = 40;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const DISCOUNT_7_MORE_DAYS = 50;
+  const DISCTOUNT_3_MORE_DAYS = 20;
```

Comment:
There is a typo in the constant name 'DISCTOUNT_3_MORE_DAYS'. It should be 'DISCOUNT_3_MORE_DAYS' to maintain clarity and avoid confusion.

Reply:
Fixed